js/ui/drop_down_button.js

```diff
@@ -451,12 +449,26 @@ const DropDownButton = Widget.inherit({
 
     _popupShowingHandler() {
         this.option('opened', true);
-        this.setAria({
-            expanded: true,
-            owns: this._popupContentId
+        this._setAriaExpanded(true);
+    },
+
+    _setAriaExpanded(value) {
+        const $ariaElements = this._getExpandedAriaElements();
+        $ariaElements.forEach((ariaElement) => {
+            this.setAria({
+                expanded: value,
+                owns: value ? this._popupContentId : undefined,
+            },
+            $(ariaElement));
         });
     },
 
+    _getExpandedAriaElements() {
+        const buttonElements = this._buttonGroup.$element().find('.dx-button').toArray();
```

Comment:
Maybe we can cache `$buttonElements` in `_renderButtonGroup` to improve performance?